src/pages/index.tsx

```diff
@@ -2,5 +2,5 @@ import React from 'react';
 import { Redirect } from '@docusaurus/router';
 
 export default function Home(): JSX.Element {
-  return <Redirect to="/docs/intro" />;
+  return <Redirect to="/docs/surrealdb/intro" />;
```

Comment:
Should this not be:
```suggestion
  return <Redirect to="/surrealdb/intro" />;
```
?

Reply:
No because the base url on surrealdb.com will still be /docs. It's relative to the browsers url